modules/Microsoft.DBforPostgreSQL/flexibleServers/.test/private/deploy.test.bicep

```diff
@@ -0,0 +1,99 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.dbforpostgresql.flexibleservers-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
```

Comment:
Same as above for `serviceshort`. For the 3 letters test suffix I'd suggest `pvt` instead of `pv`

Reply:
done